autoload/airline/extensions/tabline/buffers.vim

```diff
@@ -3,7 +3,14 @@
 
 scriptencoding utf-8
 
-let s:spc = g:airline_symbols.space
+let s:spc = get(g:, 'airline#extensions#tabline#pad_with_spaces', 1)
```

Comment:
I am not convinced we really need a new configuration variable `airline#extensions#tabline#pad_with_spaces`? And it isn't documented in the help.

Reply:
By default, vim-airline adds spaces around the buffer names. I've noticed when I have a lot of buffers open with small names this means a lot of wasted space, and with powerline symbols you don't really need them (the powerline symbols are already good enough separators).

That was the reason I added the config - if you don't use powerline symbols, you probably want the spaces. Otherwise, they are unnecessary. Some people might like the spaces - I do not.

What do you suggest instead? I'm open to suggestions, of course.

Reply:
okay. Then please document this in the help document below the tabline extension.